ocean_lib/ocean/ocean_assets.py

```diff
@@ -334,6 +334,24 @@ def create(
 
         return asset
 
+    def update_trusted_algorithms_shallow(
+        self, asset: Asset, publisher_wallet: Wallet, trusted_algorithms: list = None
+    ):
+        compute_service_descriptor = asset.get_service(ServiceTypes.CLOUD_COMPUTE)
+        assert (
+            compute_service_descriptor
+        ), "update trusted algorithms failed, compute service does not exist."
+
+        if compute_service_descriptor:
+            if trusted_algorithms:
+                trusted_algorithms_list = create_publisher_trusted_algorithms(
+                    self, trusted_algorithms=trusted_algorithms
+                )
+                compute_service_descriptor[1]["attributes"]["main"]["privacy"] = {
+                    "publisherTrustedAlgorithms": trusted_algorithms_list
+                }
+        self.update(asset=asset, publisher_wallet=publisher_wallet)
```

Comment:
Not sure here, when I call `update` function if I used the correct object for `asset`. Do I need to use `self` for asset field instead of the parameter `asset`?